### PR TITLE
Update Custom code section with packages list

### PR DESCRIPTION
**Pull request by ranvijayj**
Update Custom code section with packages list
Add default & custom packages
ADHOC

### CODE REVIEW

File: docs/bot-builder/advanced/integrating-custom-code.md [hunk 1]
```diff
@@ -394,6 +394,26 @@ def main(event, context):
...
+chardet-3.0.4 
+gspread-3.1.0 
+httplib2-0.13.1 
+idna-2.8 
+mysqlclient-1.4.2 
+oauth2client-4.1.3 
+pyasn1-0.4.6 
+pyasn1-modules-0.2.6 
+pytz-2019.2 
+redis-3.2.1
```
Comment (tejas-haptik) [inline docs/bot-builder/advanced/integrating-custom-code.md]: Is `redis` very specific to our internal use cases? Do we want people who are not `ISV`'s but still using Code Executor to know about this?

File: docs/bot-builder/advanced/integrating-custom-code.md [hunk 2]
```diff
@@ -394,6 +394,26 @@ def main(event, context):
...
 ```
 
+##### Supported Packages 
+1. *Default* - [List of Packages here](https://gist.github.com/gene1wood/4a052f39490fae00e0c3#file-all_aws_lambda_modules_python3-6-txt)
+2. *Extra packages* -  
+chardet-3.0.4 
+gspread-3.1.0 
+httplib2-0.13.1 
+idna-2.8 
+mysqlclient-1.4.2
```
Comment (tejas-haptik) [inline docs/bot-builder/advanced/integrating-custom-code.md]: similarly for `MySql`. Should we avoid this for security reasons?

Comment (ranvijayj) [reply docs/bot-builder/advanced/integrating-custom-code.md]: I have just added this to show that we will support datastores going forward. Those methods will be different. This is a generic light library. 
No security issues.

File: docs/bot-builder/advanced/integrating-custom-code.md (hunk 1 above)
Comment (ranvijayj) [reply docs/bot-builder/advanced/integrating-custom-code.md]: Any ISV might have a Redis store which they might want to use to test. We are not providing a dedicated data store for now but will be there. 
A method like haptik_data_store.
Eventually, we want people to know.

File: docs/bot-builder/advanced/integrating-custom-code.md (hunk 2 above)
Comment (ranvijayj) [reply docs/bot-builder/advanced/integrating-custom-code.md]: I could actually remove these, for now, Redis and SQL and add later only. 
But with these packages, no one will be able to access our current data stores anyways but could test out with other endpoints.

Comment (GanadiniAkshay) [reply docs/bot-builder/advanced/integrating-custom-code.md]: I think its fine. These are just libraries that enable them to use their own datastores (sql or redis) which will be needed.

**Pull request merged**